CONTRIBUTING.md

```diff
@@ -36,7 +36,8 @@ Please see our [Contributor Code of Conduct](https://github.com/ionic-team/stenc
    [Install it](https://docs.volta.sh/guide/getting-started) before proceeding.
    1. There's no need to install a specific version of npm or Node right now, it shall be done automatically for you in
       the next step
-5. Run `npm install`
+5. Run `npm ci`
+6. (Optional) If you are working on [Jest support](./src/testing/jest), [see the installation steps in that directory's README](./src/testing/jest/README.md#installing-dependencies).
```

Comment:
would it make sense for us to put this in a `postinstall` script in our root `package.json` so the jest deps are just sort of always there for us?

Reply:
I don't think so, as it breaks the `npm pack` workflow of testing a Stencil tarball.
When installing a tarball of an npm package (so one where we generate via `npm ci && npm run build && npm pack`) then go to `npm install [PATH_TO_TARBALL]` in a starter project (or wherever), the post install scripts will attempt to run in the project we're installing the tarball into :-(

Reply:
ohh right I forgot that they would run then too

hmmmmmm wondering if maybe we want a `bootstrap` command at some point 🤔 

something to worry about later!